### PR TITLE
Add multi-monitor support

### PR DESCRIPTION
**Pull request by roscale**

### CODE REVIEW

**Comment by roscale** [comment]
This PR should get much smaller once #20 is merged.

Also, don't merge this yet because I still have to test it on my laptop, that's why I left it in draft. You can review it though.

File: embedder/src/drm_backend.rs
Comment (PapyElGringo) [inline embedder/src/drm_backend.rs:321]: I do have a multi-gpu setups if needed for testing

**Review by PapyElGringo — APPROVED** [review]

File: embedder/src/drm_backend.rs
Comment (roscale) [reply embedder/src/drm_backend.rs:321]: Ok, maybe in the future when this becomes a priority.

**Pull request merged**
